### PR TITLE
Add tritimap recipes

### PR DESCRIPTION
**Pull request by fei0810**
Add Triti-Map to Bioconda
Triti-Map: A Snakemake-based pipeline for gene mapping in Triticeae.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/tritimap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{% set version = "0.9.0" %}
+{% set sha256 = "c3a0249f4622fe7b1f4d2d6b716dfb0dd5a9bd6414ac2669ebc3239d50855c58" %}
+
+
+package:
+  name: tritimap
+  version: {{ version }}
+
+source:
+  url: https://github.com/fei0810/Triti-Map/archive/{{ version }}.tar.gz
```
Comment (xuzhougeng) [inline recipes/tritimap/meta.yaml:10]: https://github.com/fei0810/Triti-Map/archive/{{ version }}.tar.gz should replace to
https://github.com/fei0810/Triti-Map/archive/v{{ version }}.tar.gz

**Comment by xuzhougeng** [comment]
@BiocondaBot please add label

File: recipes/tritimap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  summary: Triti-Map is a Snakemake-based pipeline for gene mapping in Triticeae.
+  description: |
+    Triti-Map is a Snakemake-based pipeline for gene mapping in Triticeae, which contains a suite of user-friendly computational packages and web-interface integrating multi-omics data from Triticeae species including genomic, epigenomic, evolutionary and homologous information.
+    
+    Triti-Map could efficiently explore trait-related genes or functional elements not present in the reference genome and reduce the time and labor required for gene mapping in large genome species.
+
+extra:
+  recipe-maintainers:
+    - Fei Zhao
+    - Zhougeng Xu
```
Comment (dpryan79) [inline recipes/tritimap/meta.yaml:67]: Can you make these github ids (e.g., fei0810)? Then you'll be notified to approve automatically generated update PRs.

Comment (fei0810) [reply recipes/tritimap/meta.yaml]: Hi, Ryan. I have changed the recipe-maintainers IDs.

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**